tests/csm/rest/test_capacity_quota.py

```diff
@@ -641,3 +644,398 @@ def test_40640(self):
+
+            assert_utils.assert_equals(self.user_id, uid, "id is not equal")
+            assert_utils.assert_equals(total_objects, t_obj, "Number of objects not equal")
+            assert_utils.assert_equals(total_objects, num_objects, "Number of objects not equal")
+            assert_utils.assert_equal(total_size, t_size, "Total Size mismatch found")
+            assert_utils.assert_equal(total_size, data_size, "Total Size mismatch found")
+            assert_utils.assert_greater_equal(total_size, m_size, "Total Used Size mismatch found ")
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41954')
+    def test_41954(self):
+        """
+        Test GET capacity stats for delete objects using admin users
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_41954"]
+        available_size = test_cfg["max_size"]
+        random_size = self.csm_obj.random_gen.randrange(1,available_size)
+        num_objects = math.floor(available_size / random_size)
+        data_size = num_objects * random_size
+        self.log.info("Step 1: Create N objects of Random size totals to S bytes")
+        for _ in range(0, num_objects):
+            resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
+                                              self.akey, self.skey, object_size=random_size)
+            assert resp, "Put object Failed"
+        self.log.info("Step 3: Get capacity count from AWS")
+        total_objects, total_size = s3_misc.get_objects_size_bucket(self.bucket,
+                         self.akey, self.skey)
+
+        resp = self.awscli_s3api_obj.list_objects_v2(self.bucket)
+        assert resp[0], resp[1]
+        for obj in resp:
+            self.log.info("Step 3: Delete object: %s", obj)
+            assert s3_misc.delete_object(
+                self.bucket, obj, self.akey, self.skey), "Failed to delete bucket."
+            self.log.info("Step 4: Perform GET API to get capacity usage")
+            resp = self.csm_obj.get_capacity_usage("user", self.user_id)
+            assert resp.status_code == HTTPStatus.OK, \
+                "Status code check failed for get capacity"
+        self.log.info("Step 5: Perform GET API to get capacity usage after all"
+                      "objects are deleted")
+        resp = self.csm_obj.get_capacity_usage("user", self.user_id)
+        assert resp.status_code == HTTPStatus.OK, \
+               "Status code check failed for get capacity"
+        self.log.info("Step 3: Get capacity count from AWS")
+        total_objects, total_size = s3_misc.get_objects_size_bucket(self.bucket,
+                         self.akey, self.skey)
+
```

Comment:
Final GET capacity check that everything is 0

Reply:
added

Reply:
added